src/_pytest/outcomes.py

```diff
@@ -55,13 +57,13 @@ class Skipped(OutcomeException):
 
     def __init__(
         self,
-        msg: Optional[str] = None,
+        reason: Optional[str] = None,
```

Comment:
for consistency with base classes, lets leave the msg name here

we ought to consider deprecation/removal of msg alltogether as exception attibute, but thats for much later